packages/web-components/src/text/text.styles.ts

```diff
@@ -36,17 +36,22 @@ import {
 export const styles = css`
   ${display('inline')}
 
+  ${/* slot defaults display to contents, this ensures slotted items inherit from the host */ ''}
+  slot {
+    display: inherit;
+  }
+
```

Comment:
I would _really_ prefer to not do this if we can avoid it - it's definitely not ideal. Do we actually need inheritance in this case?

What about simply using a selector list for block to make sure both get updated?

```css
  :host([block]),
  :host([block]) ::slotted(*) {
    display: block;
  }
```